package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "generator-jhipster",
-  "version": "7.1.0",
+  "version": "7.1.1",
```

```suggestion
  "version": "7.1.0",
```
Versions are bumped at release

---

@mshima  When I changed it back to 7.1.0, it removes the new files that are marked to be removed if the version is < 7.1.1. I'm not sure if that remove files section in the `cleanup.js` file (lines 490-495) is even necessary since Couchbase is only supported from the next version onwards. That section was already in the fork I was working from, so I kept it in there, but since it seems to be causing issues - is it safe to remove?

---

Removed files needed to be updated.
This is used for updating from a jhipster version that couchbase wasn't broken.